data-serving/data-service/src/controllers/preprocessor.ts

```diff
@@ -102,58 +101,82 @@ export const setBatchUpsertRevisionMetadata = async (
             ]),
     );
 
-    // Get sourceId:sourceEntryId for cases that are included in the batch
-    // request but that won't be modified (because the request case is identical
-    // to the existing one).
-    const unchangedCaseIdSet = request.body.cases.reduce(
-        // TODO: Type request Cases.
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        (unchangedIds: Set<string>, c: any) => {
-            if (c.caseReference?.sourceId && c.caseReference?.sourceEntryId) {
-                const existingCase = existingCasesByCaseRefCombo.get(
-                    c.caseReference.sourceId +
-                        ':' +
-                        c.caseReference.sourceEntryId,
-                );
-                if (existingCase !== undefined && existingCase.equalsJSON(c)) {
-                    unchangedIds.add(existingCase._id.toString());
-                }
+    const unchangedCaseIdSet = new Set();
```

Comment:
Is this used, anymore?

Reply:
It was used in future middleware to drop unchanged cases before calling `Case.insertMany`. Removed both.